.changeset/wise-flies-doubt.md

```diff
@@ -0,0 +1,13 @@
+---
+'@leafygreen-ui/hooks': patch
+---
+
+Removes `useMemo` from `useAvailableSpace`, which was preventing the hook from returning the correct value.
+
+Previously, `useViewportSize` returned `null` initially, then re-rendered with the correct viewport size. However, with the recent changes, `useViewportSize` now immediately returns the correct viewport size. This caused issues in the `useAvailableSpace` hook, as the value returned from `useViewportSize` is a dependency for `useAvailableSpace`.
```

Comment:
I'll update this so that it's clearer but on first render, the ref is not available but the viewport size is. When the menu is opened, the ref is available but the viewport size remains the same so the memo callback is not triggered since refs do not trigger a rerender.

Previously, the `useViewportSize` would return null, then rerender, which would trigger the memo callback.